bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "purescript-decimals",
+  "version": "1.0.0",
```

Comment:
I think it's not necessary to provide `version` in `bower.json`, we can rely on git tags.